options/locale/locale_nl-NL.ini

```diff
@@ -681,6 +681,7 @@ archive.issue.nocomment=Deze repo is gearchiveerd. U kunt niet reageren op probl
 archive.pull.nocomment=Deze repo is gearchiveerd. U kunt niet reageren op pull requests.
 
 form.reach_limit_of_creation=U hebt uw limiet van %d repositories al bereikt.
+form.reach_limit_of_private_creation=U hebt uw limiet van %d prive repositories al bereikt.
```

Comment:
for languages that aren't english our CI will overwrite them with the translations from crowdin, when this is merged please submit them there :)

Reply:
Check... adding them there now.

Reply:
I thought so, I've just been adding to the Dutch translation for now.